### PR TITLE
Add testing & code review sections to UI README.md

### PR DESCRIPTION
**Pull request by cuongdo**
Preview the change here:

https://github.com/cuongdo/cockroach/blob/add_to_ui_readme/ui/README.md

cc @mrtracy

---

This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/5290)

### CODE REVIEW

File: ui/README.md [hunk 1]
```diff
@@ -74,3 +74,40 @@ Be sure to commit any changes to `npm-shrinkwrap.json`.
...
+
+Checking your work with different browser window dimensions is especially
+important, because we don't yet have a responsive admin UI design.
+
+## Reviewing UI changes
+
+We really should have a UI style guide when the UI code is more settled.
+
+In addition to reviewing for correctness and consistent, idiomatic use of
+TypeScript and the various UI frameworks we use, pay particular attention to
```
Comment (tamird) [inline ui/README.md]: missing oxford comma

**Comment by cuongdo** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[ui/README.md, line 105 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/5290#-KCyy6ytZy-SGDBDH99U-r1-105:-KCzeNZKIPkwsstAUwAQ:-1363311750) ([raw file](https://github.com/cockroachdb/cockroach/blob/ba4799cf8e6e135f4316e1de8e82846e2cf31bd8/ui/README.md#L105)):*
It doesn't apply here as it's `correctness` and `consistent, idiomatic use of TypeScript and the various UI frameworks we use` -- just two items.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/5290)*

**Comment by tamird** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[ui/README.md, line 105 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/5290#-KCyy6ytZy-SGDBDH99U-r1-105:-KCzeqgyZ6LEV7H1Fs1C:559151627) ([raw file](https://github.com/cockroachdb/cockroach/blob/ba4799cf8e6e135f4316e1de8e82846e2cf31bd8/ui/README.md#L105)):*
I stand corrected.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/5290)*

**Comment by maxlang** [comment]
I'm not sure if this is the right place for this. We definitely need to do visual regression testing, and run through a UI QA script, but the elements mentioned here seem too specific. I'd rather we have a script we run through every evening or once a week that checks for all the issues we know we've seen (like wrapping visualizations.) Even if we ask everyone to go to all the different pages and run through a QA script, issues will fall through the cracks.

What about creating QA.md file and putting our QA actions there? Then for any big change we can ask someone to run through the script or a portion of the script, and we can also run through it ourselves occasionally to catch any issues that went unnoticed?

Otherwise I agree with the ```Reviewing Your Change``` section.

**Comment by tamird** [comment]
@cuongdo @maxlang can we close this up, one way or another?

**Comment by cuongdo** [comment]
Looks like this'll need to be rewritten for the UI next work. CC @mrtracy & @maxlang

**Pull request closed without merging**